repos/core-setup.proj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <EnvironmentVariables Include="OfficialBuildId=$(OfficialBuildId)" />
+    <EnvironmentVariables Include="PackageVersionStamp=rc1" />
```

Comment:
Is this enough?  Do you have to do something with StabilizePackageVersion as well?

Reply:
Yes, I think this is closer to what prodcon does so probably better to do it this way.

Reply:
Prodcon actually sets the PB_IsStable and PB_VersionStamp as env variables/build parameters and then the repo's map them to what they use. See https://github.com/dotnet/corefx/blob/master/buildpipeline/pipeline.json#L23 for example. While the core-repos are pretty consistent the others likely don't use those same properties.